Cargo.toml

```diff
@@ -61,7 +62,7 @@ error-stack = { version = "0.5", default-features = false, features = [
 fastimer = { version = "0.9.0" }
 fastrace = { version = "0.7.9" }
 fastrace-opentelemetry = { version = "0.10.0" }
-foyer = { version = "0.16.0", features = ["nightly"] }
+foyer = { version = "0.17", features = ["nightly", "tracing"] }
```

Comment:
@leiysky do we enable foyer tracing for Percas case?

Reply:
We haven't enabled tracing for Percas so far. Percas is a service with heavy accessing, we should sample the trace spans down to a reasonable number.

Reply:
OK. Then I'd remove `tracing` feature for now. Pushed a new commit.